client/my-sites/plugins/plugins-browser-list/test/index.jsx

```diff
@@ -25,29 +20,38 @@ const plugins = [
 const props = {
 	plugins,
 	variant: PluginsBrowserListVariant.Fixed,
-	listName: 'woocommerce',
-	title: 'woocommerce',
-	subtitle: '100 plugins',
+	listName: 'arbitrary-plugin-list-name',
+	title: 'arbitrary-plugin-list-title',
+	subtitle: 'arbitrary-plugin-list-subtitle',
 	site: {
 		plan: PLAN_FREE,
 	},
 	size: 6,
 };
 
+const queryPlaceholders = () => {
+	const listItems = screen.queryAllByRole( 'listitem' );
+	return listItems.filter( ( item ) => Array.from( item.classList ).includes( 'is-placeholder' ) );
```

Comment:
Any reason why we're not using [`DOMTokenList.contains()`](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/contains) here?

```suggestion
	return listItems.filter( ( item ) => item.classList.contains( 'is-placeholder' ) );
```

Reply:
Good call 👍🏻